ansible_collections/arista/avd/molecule/eos_designs_deprecated_vars/inventory/host_vars/port-channel-short-esi.yml

```diff
@@ -0,0 +1,22 @@
+type: l2leaf
+l2leaf:
+  nodes:
+    - name: port-channel-short-esi
+
+servers:
+  - name: server01_ESI
+    rack: RackC
+    adapters:
+      - endpoint_ports: [ Eth1, Eth2]
+        switch_ports: [ Ethernet10, Ethernet10]
+        switches: [ port-channel-short-esi, Leaf1]
+        profile: TENANT_A_B
+        port_channel:
+          description: PortChanne1
+          mode: active
+          short_esi: '0303:0202:0101'
+          subinterfaces:
+            - number: 111
+              short_esi: 0303:0202:0111
+            - number: 222
+              short_esi: 0303:0202:0222
```

Comment:
These are not deprecated yet, so they should not be in the `deprecated_vars` scenario.

Reply:
removed it